numba/core/codegen.py

```diff
@@ -714,7 +714,8 @@ def _pass_manager_builder(self):
         missed...
         """
         pmb = lp.create_pass_manager_builder(
-            opt=config.OPT, loop_vectorize=config.LOOP_VECTORIZE)
+            opt=config.OPT, loop_vectorize=config.LOOP_VECTORIZE,
+            slp_vectorize=True)
```

Comment:
Shouldn't this also get a `config` flag? e.g. `config.SLP_VECTORIZE`

Reply:
Can do, but I really want to move away from specific config flags for each feature in the optimiser, I think it should be in the hands of the user but also it shouldn't be spelled one at a time like this. For 0.52 shall we do a flag so it's easy to debug/turn off but this is the last one like it!?!

Reply:
Ok, make it the last flag.